tests/functional/spec/components/radio/radio.spec.js

```diff
@@ -0,0 +1,177 @@
+const helpers = require('../../../helpers');
+
+describe('Component: Radio', function() {
+
+describe('Given I start a Mandatory Radio survey', function() {
+
+  const RadioMandatoryPage = require('../../../pages/components/radio/mandatory/radio-mandatory.page');
+  const RadioSummaryPage = require('../../../pages/components/radio/mandatory/summary.page');
+  var radio_schema = 'test_radio_mandatory.json';
+
+  before('load the survey', function() {
+    return helpers.openQuestionnaire(radio_schema);
+    });
+
+  it('When I submit data it should be persisted and Then displayed on the summary', function() {
```

Comment:
i think this scenario could be better written. Something like:
`When I have selected a radio option, Then the selected option should be displayed in the summary`

Reply:
maybe along these lines?